modules/core/src/lib/effect-manager.ts

```diff
@@ -16,9 +17,24 @@ export default class EffectManager {
     this._setEffects([]);
   }
 
-  addDefaultEffect(effect: Effect) {
-    if (!this._defaultEffects.find(e => e.constructor === effect.constructor)) {
-      this._defaultEffects.push(effect);
+  // TODO - better Effect API, may be cleaner to add `order` to `Effect` class
+  /**
+   * Register a new default effect, i.e. an effect present regardless of user supplied props.effects
+   */
+  addDefaultEffect(
+    /** Effect instance. The same effect can only be added once. */
+    effect: Effect,
+    /** Used to sort effects. In case of conflict, effects will be used in the order of registration. */
+    order: number = Infinity
```

Comment:
Is the order pretty clearly defined by the effects, such that users normally wouldn't need to override it? I.e. Would it make sense to have order in the effects themselves, so they just sorted themselves appropriately when added?